product_docs/docs/pgd/5/reference/testingandtuning.mdx

```diff
@@ -4,117 +4,116 @@ navTitle: Testing and tuning
 indexdepth: 2
 ---
 
-EDB Postgres Distributed has tools which help with testing and tuning of your PGD clusters. For background, read the [Testing and Tuning](../testingandtuning) section.
+EDB Postgres Distributed has tools that help with testing and tuning your PGD clusters. For background, see [Testing and tuning](../testingandtuning).
 
 
-## `pgd_bench`
+## pgd_bench
 
 ### Synopsis
 
-A benchmarking tool for PGD enhanced PostgreSQL.
+A benchmarking tool for PGD-enhanced PostgreSQL.
```

Comment:
I'd like this to say EDB Postgres Distributed deployments

Reply:
```suggestion
A benchmarking tool for EDB Postgres Distributed deployments.
```